app/locales/lt/translations.js

```diff
@@ -100,1007 +100,1007 @@ export default {
     "97": "97",
     "98": "98",
     "99": "99",
-    "Key": "Lithuanian Translation",
+    "Key": "Italian Translation",
```

Comment:
This appears to be the translation for italian ([it](https://github.com/abought/isp/blob/4e53b1ea4d910ee28800bae9d6c3d0dc019bb2e4/app/components/language-picker/countries.js#L270)), but it has been copied into the translation file for lithuanian ([lt](https://github.com/abought/isp/blob/4e53b1ea4d910ee28800bae9d6c3d0dc019bb2e4/app/components/language-picker/countries.js#L310-L310)). Can you double-check this and make sure these files contain the intended translations?